feelpp/feel/feelmesh/meshsupport.hpp

```diff
@@ -363,18 +347,124 @@ private :
 private :
     mesh_ptrtype M_mesh;
     range_elements_type M_rangeElements;
-    range_faces_type M_rangeInterProcessFaces;
-    range_faces_type M_rangeBoundaryFaces;
-    range_faces_type M_rangeInternalFaces;
-    std::unordered_set<size_type> M_rangeMeshElementsIdsPartialSupport;
-    std::unordered_set<size_type> M_rangeMeshElementsGhostIdsPartialSupport;
+    mutable range_faces_type M_rangeInterProcessFaces;
+    mutable range_faces_type M_rangeBoundaryFaces;
+    mutable range_faces_type M_rangeInternalFaces;
+    std::unordered_map<flag_type,range_faces_type> M_rangeMarkedFaces;
```

Comment:
The ranges of marker shouldn't be saved, this can be evolved and can be taken memory. You must use the same way of the classic filter (without mesh support) just iterate over internal and boundary faces

Reply:
which data member are you talking about ? `M_rangeMarkedFaces` or the previous ones as well (they were already stored) ?

Reply:
M_rangeMarkedFaces, the others one (boundary,internal,interprocesses) are necessary and constants